config/application.rb

```diff
@@ -0,0 +1,13 @@
+require_relative "boot"
+
+require "rails/all"
+
+Bundler.require(*Rails.groups)
+
+module WellAcademy
+  class Application < Rails::Application
+    config.load_defaults 7.2
```

Comment:
why we are using load_defaults?

Reply:
Bhai, when upgrading Rails, using `config.load_defaults `helps our app smoothly transition to newer Rails defaults. We don’t need to manually configure many aspects of our app. It helps with backward compatibility and smoother upgrades.